source/rust_verify/src/main.rs

```diff
@@ -124,6 +124,44 @@ pub fn main() {
 
     let pervasive_path = our_args.pervasive_path.clone();
 
+    if our_args.error_report {
+        let mut args: Vec<String> = std::env::args().collect();
+        args.remove(0);
+
+        let index = args.iter().position(|x| *x == "--error-report").unwrap();
+        args.remove(index);
+
+        // error_report binary will always add --time flag
+        if let Some(index) = args.iter().position(|x| *x == "--time") {
+            args.remove(index);
+        }
```

Comment:
Copying over comments from https://github.com/verus-lang/verus/pull/612#discussion_r1254769157 which is now marked as outdated:

@jaybosamiya:
> This probably should be handled by the `error_report` binary?

@ahuoguo:
> I removing `--error-report` flag at the same time, so these two might want to stay in the same level?

@jaybosamiya:
> The intention with placing the `--time` removal in the `error_report` binary is that we want to report (to the `error_report` binary) something that fully captures the command the user used for Verus, as closely as possible; the fact that an `--error-report` flag was passed is only a way to invoke the `error_report` binary; but whether the user _explicitly_ passed in `--time` or not _may_ potentially be something that we might want/need to actually track (for `--time` I guess it is not _as_ crucial, but hopefully you see my reasoning on why the two flags are somewhat different in their removal)